src/Microsoft.DotNet.Watcher.Tools/CommandLineOptions.cs

```diff
@@ -44,8 +51,15 @@ public static CommandLineOptions Parse(string[] args, TextWriter consoleOutput)
                 return null;
             }
 
+            if (optQuiet.HasValue() && optVerbose.HasValue())
+            {
+                consoleOutput.WriteLine("Found both --quiet and --verbose options. The --verbose flag will be ignored.".Bold().Black());
+            }
+
             return new CommandLineOptions
             {
+                IsQuiet = optQuiet.HasValue(),
+                IsVerbose = optVerbose.HasValue() && !optQuiet.HasValue(), // --quiet takes precedence
```

Comment:
Should it be an error case instead? You can't be noisy and quite at the same time so show an error saying what the problem is.